crates/cext/src/sparse_observable.rs

```diff
@@ -286,9 +286,9 @@ pub unsafe extern "C" fn qk_obs_add_term(
 /// ```c
 ///     QkObs *obs = qk_obs_identity(100);
 ///     QkObsTerm term;
-///     int exit_code = qk_obs_term(obs, 0, &term);
+///     QkExitCode exit_code = qk_obs_term(obs, 0, &term);
 ///     // out-of-bounds indices return an error code
-///     // int error = qk_obs_term(obs, 12, &term);
+///     // QkExitCode error = qk_obs_term(obs, 12, &term);
 /// ```
```

Comment:
You fixed indentation in other places - here as well?

Reply:
that's fine no worries, it's why I left the "approve" but didn't hit merge, so you could choose